lib/pry/prompt.rb

```diff
@@ -94,11 +94,10 @@ def prompt_name(name)
       end
     end
 
-    add(:default, <<DESC) do |context, nesting, _pry_, sep|
-The default Pry prompt. Includes information about the current expression
-number, evaluation context, and nesting level, plus a reminder that you're
-using Pry.
-DESC
+    add 'default',
+        'The default Pry prompt. Includes information about the current expression ' \
+        'number, evaluation context, and nesting level, plus a reminder that you\'re ' \
```

Comment:
No need to escape `'` if you can use double quotes.

Reply:
fixed